src/features/campaigns/l10n/messageIds.ts

```diff
@@ -77,6 +77,9 @@ export default makeMessages('feat.campaigns', {
     heading: m('Feedback and Surveys (none configured)'),
   },
   form: {
+    createAreaAssignment: {
+      defaultQuestion: m('Did you complete the mission?'),
```

Comment:
Is this the time to replace "mission", which feels so "big". Or am I the only one who feels that way? I would prefer "assignment" or "task", or something like that.